sys/net/crosslayer/ng_pktbuf/_pktbuf_static.c

```diff
@@ -165,7 +168,7 @@ void *_pktbuf_internal_alloc(size_t size)
 
     while ((node->next != NULL)
            /* and if space between current and next allocation is not big enough */
-           && ((_start_idx(node->next) - _end_idx(node)) < _total_sz(size))) {
+           && ((_start_idx(node->next) - _end_idx(node)) < _al_sz(_total_sz(size)))) {
```

Comment:
This function we are talking in: `_pktbuf_internal_alloc()`. The node is "created" in https://github.com/RIOT-OS/RIOT/pull/3316/files#diff-2901d789a30fdc0411aca7f3e3ec7249R176, but only filled after it is determined, that there is enough space available. (https://github.com/RIOT-OS/RIOT/pull/3316/files#diff-2901d789a30fdc0411aca7f3e3ec7249R183 and following)